app/src/main/java/com/example/fittracker/MainActivityView.java

```diff
@@ -0,0 +1,46 @@
+package com.example.fittracker;
+
+import android.app.Activity;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class MainActivityView implements LogInContract.View {
+    @BindView(R.id.mail_usuario) EditText mail;
+    @BindView(R.id.password) EditText password;
+    private Activity actividad;
```

Comment:
use a WeakReference here inseat of the Activity.
private WeakReference<Activity> activityRef;

and to use it you will have to do activityRef.get()
or make a method like

public Activity getActivity() {
        return activityRef.get();
    }

Reply:
Done